internal/utils/main.go

```diff
@@ -0,0 +1,32 @@
+package utils
+
+import (
+	"encoding/hex"
+	"errors"
+	"time"
+
+	"github.com/stellar/go/xdr"
+)
+
+//PanicOnError is a function that panics if the provided error is not nil
```

Comment:
I like the instincts, but I'm not actually sure this is the best practice in Go. @leighmcculloch I feel like you'd know the answer to this.